### PR TITLE
Attempting to add note on there being no ReprMan readings

### PR DESCRIPTION
**Pull request by jessicabartley**
attempting to replace the auto "Readings (to be added)" text with text that reads "No readings for this week's ReproNim elcture". Not sure if my edits match the needed format to make this change.

### CODE REVIEW

File: _posts/2021-02-05-week-10.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ repronim_title: ReproMan – Execution and Environment Manager
 repronim_video: https://www.youtube.com/embed/grIVFbYH7YE
 repronim_slides:
 repronim_reading_links:
-repronim_reading_names: No readings for this week's ReproNim lecture!
+repronim_reading_names: [No readings for this week's ReproNim lecture!]
```
Comment (tsalo) [inline _posts/2021-02-05-week-10.md:18]:
```suggestion
repronim_reading_names: ["No readings for this week's ReproNim lecture!"]
```

File: _posts/2021-02-05-week-10.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ repronim_title: ReproMan – Execution and Environment Manager
 repronim_video: https://www.youtube.com/embed/grIVFbYH7YE
 repronim_slides:
 repronim_reading_links:
```
Comment (tsalo) [inline _posts/2021-02-05-week-10.md:17]:
```suggestion
repronim_reading_links: ["https://abcd-repronim.github.io/materials/week-10/"]
```
Link back to itself.

**Review by tsalo — CHANGES_REQUESTED** [review]
If we don't want to make things very complicated, you'll need a URL for the hyperlink. You can just put the materials page though...

File: _posts/2021-02-05-week-10.md (hunk 2 above)
Comment (richford) [reply _posts/2021-02-05-week-10.md:17]: Why link back to itself when you can have a [cat gif](https://media.giphy.com/media/1iu8uG2cjYFZS6wTxv/giphy.gif)?

Comment (tsalo) [reply _posts/2021-02-05-week-10.md:17]: okay yes that's objectively better

Comment (jessicabartley) [reply _posts/2021-02-05-week-10.md:17]: omg. how can I _not_ use that?

**Review by jessicabartley — COMMENTED** [review]
good to go!

**Pull request merged**